src/api/app/assets/stylesheets/webui2/personal-navigation.scss

```diff
@@ -0,0 +1,18 @@
+#personal-navigation {
+    & .nav-item a {
+        font-size: 0.9rem;
+        color: $main_green;
```

Comment:
These links are the opposite of the other links (switching from green to blue). I would do just like the others.

Reply:
No, this is intended. It is like we have in the non bootstrap main page